dll/win32/imm32/ime.c

```diff
@@ -6,7 +6,7 @@
  *              Copyright 2002, 2003, 2007 CodeWeavers, Aric Stewart
  *              Copyright 2017 James Tabor <james.tabor@reactos.org>
  *              Copyright 2018 Amine Khaldi <amine.khaldi@reactos.org>
- *              Copyright 2020 Oleg Dubinskiy <oleg.dubinskij2013@yandex.ua>
+ *              Copyright 2020 Oleg Dubinskiy <oleg.dubinskij30@gmail.com>
```

Comment:
Maybe it is enough to have 3 maintainers listed here, instead of 4. Just a thought. To reduce future grooming efforts.